### ISSUE TITLE
System Error when running Ganache 2.0.0-beta.2 on darwin

### ISSUE THREAD

**Issue by zulhfreelancer** [trufflesuite/ganache#1111]
PLATFORM: darwin
GANACHE VERSION: 2.0.0-beta.2

EXCEPTION:
```
Error: The module '/Users/zulh/go/src/hg_blockchain/node_modules/scrypt/build/Release/scrypt.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 59. This version of Node.js requires
NODE_MODULE_VERSION 57. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).
    at process.module.(anonymous function) [as dlopen] (ELECTRON_ASAR.js:172:20)
    at Object.Module._extensions..node (module.js:598:18)
    at Object.module.(anonymous function) [as .node] (ELECTRON_ASAR.js:172:20)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/scrypt/index.js:3:20)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/scrypt/index.js:388:3)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/scrypt.js/node.js:1:169)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/scrypt.js/node.js:9:3)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/web3-eth-accounts/src/index.js:35:16)
    at Object.<anonymous> (/Users/zulh/go/src/hg_blockchain/node_modules/web3-eth-accounts/src/index.js:529:3)
```

**Comment by nikita-fuchs** [comment]
`npm uninstall scrypt`
`npm install scrypt --save`

should do the trick. This thing is super annoying.

**Comment by DiscRiskandBisque** [comment]
@zulhfreelancer is this coming from a binary or running Ganache directly from the repo?

**Comment by zulhfreelancer** [comment]
@DiscRiskandBisque I installed the DMG file from the Releases page.

**Comment by davidmurdoch** [comment]
Should be fixed in 2.0.0 by #1086.
